docs/learn-treasury.md

```diff
@@ -2,24 +2,72 @@
 
 Proposals may consist of (but are not limited to):
 
- - Infrastructure deployment and continued operation.
- - Network security operations (monitoring services, continuous auditing).
- - Ecosystem provisions (collaborations with friendly chains).
- - Marketing activities (advertising, paid features, collaborations).
- - Community events and outreach (meetups, pizza parties, hackerspaces).
- - Software development (wallets and wallet integration, clients and client upgrades).
+- Infrastructure deployment and continued operation.
+- Network security operations (monitoring services, continuous auditing).
+- Ecosystem provisions (collaborations with friendly chains).
+- Marketing activities (advertising, paid features, collaborations).
+- Community events and outreach (meetups, pizza parties, hackerspaces).
+- Software development (wallets and wallet integration, clients and client upgrades).
 
-The treasury is ultimately controlled by the stakeholders, and how the funds will be spent is up to their judgment.
+The treasury is ultimately controlled by the council, and how the funds will be spent is up to their judgment.
 
-## Resources
+## Funding the Treasury
+
+The Treasury is funded from different sources:
+
+1. When a validator is slashed for equivocation or invalid attestations of parachain blocks, the slashed amount is sent to the Treasury with a reward going to the entity which reported the validator (another validator or fisherman). The reward is taken from the slash amount and varies per offence and number of reporters.
+2. Transaction fees paid on the Relay Chain for executing balance transfers, governance votes, council actions or any other function allowed by [pallets](https://substrate.dev/docs/en/tutorials/adding-a-module-to-your-runtime/) included in the runtime.
+3. Staking inefficiency. [Inflation](learn-staking#inflation) is designed to be 10% in the first year, and the ideal staking ratio is set at 50%, meaning half of all tokens should be locked in staking. Any deviation from this ratio will cause a proportional amount of the inflation to go to the Treasury. In other words, if 50% of all tokens are staked, then 100% of the 10% inflation goes to the validators as reward.
+4. Once [Parathreads](learn-parathreads) are live, the per-block auction they have to go through to include their block on the Relay Chain will also likely contribute to the Treasury.
+
+## Creating a Treasury Proposal
+
+The proposer has to deposit 5% of the requested amount or 20 KSM (whichever is higher) as an anti-spam measure. This amount is burned if the proposal is rejected, or refunded otherwise. These values are subject to [governance](learn-governance) so they may change in the future. The way to always find out the up to date values is to [query the constants](https://polkadot.js.org/apps/#/chainstate/constants) as in the image below, or to look at the [source code](https://github.com/paritytech/substrate/blob/master/frame/treasury/src/lib.rs#L784).
```

Comment:
We should never be linking to code from the wiki. It's an implementation detail.
```suggestion
The proposer has to deposit 5% of the requested amount or 20 KSM (whichever is higher) as an anti-spam measure. This amount is burned if the proposal is rejected, or refunded otherwise. These values are subject to [governance](learn-governance) so they may change in the future.
```